aws_lambda_powertools/utilities/parser/models/sns.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+from typing import Dict, List, Optional
+
+from pydantic import BaseModel
+from pydantic.networks import HttpUrl
+from typing_extensions import Literal
+
+
+class SqsMsgAttributeModel(BaseModel):
+    Type: str
+    Value: str
+
+
+class SnsNotificationModel(BaseModel):
```

Comment:
We need `groupId` and `dedupId` optional fields that might be added when FIFO is enabled - Haven't tried yet myself.

Publisher reference: https://docs.aws.amazon.com/sns/latest/dg/fifo-topic-code-examples.html#fifo-topic-java-publish

Reply:
Resolving it as @risenberg-cyberark confirmed Lambda cannot directly subscribe to SNS FIFO topics, it'd have to go through SQS FIFO, in which case our model already covers it.